zulipterminal/config/keys.py

```diff
@@ -323,77 +323,77 @@ class KeyBinding(TypedDict):
     'BEGINNING_OF_LINE': {
         'keys': ['ctrl a'],
         'help_text': 'Jump to the beginning of line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'END_OF_LINE': {
         'keys': ['ctrl e'],
         'help_text': 'Jump to the end of line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'ONE_WORD_BACKWARD': {
         'keys': ['meta b'],
         'help_text': 'Jump backward one word',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'ONE_WORD_FORWARD': {
         'keys': ['meta f'],
         'help_text': 'Jump forward one word',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'DELETE_LAST_CHARACTER': {
         'keys': ['ctrl h'],
         'help_text': 'Delete previous character (to left)',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'TRANSPOSE_CHARACTERS': {
         'keys': ['ctrl t'],
         'help_text': 'Transpose characters',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'CUT_TO_END_OF_LINE': {
         'keys': ['ctrl k'],
         'help_text': 'Cut forwards to the end of the line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'CUT_TO_START_OF_LINE': {
         'keys': ['ctrl u'],
         'help_text': 'Cut backwards to the start of the line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'CUT_TO_END_OF_WORD': {
         'keys': ['meta d'],
         'help_text': 'Cut forwards to the end of the current word',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'CUT_TO_START_OF_WORD': {
         'keys': ['ctrl w'],
         'help_text': 'Cut backwards to the start of the current word',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'PASTE_LAST_CUT': {
         'keys': ['ctrl y'],
         'help_text': 'Paste last cut section',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'UNDO_LAST_ACTION': {
         'keys': ['ctrl _'],
         'help_text': 'Undo last action',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
```

Comment:
What do you think about having two categories for editing, ie. one for navigation, and one for cutting/pasting/manipulation/undo? I think undo should be higher, if not top, of the latter set of ones in any case, since it's pretty useful! (even undoing `ctrl l`).

Reply:
Good to see this present, I think it helps :+1: